docs/pages/en-us/Architecture.md

```diff
@@ -5,78 +5,224 @@ title: Architecture
+The Audit library logs audit events on a per tenant, per application basis and facilitates the addition of new tenants and new applications.
 
-In order to use Auditing with an application, you must first specify the audit events that the application uses, and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file.
+In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch.
 
-After you author the audit event definition file, you can use it in two ways:
-
-1. To generate an application-specific, client-side auditing library
-2. To register the application with the Auditing Management web service
-
-After you generate an application-specific, client-side auditing library, the application uses the Java library to send audit event messages to the Apache Kafka messaging service.
-
-On the server-side, with the Audit Management web service API, the application's audit event definition file is used to create an audit event schema for the application within Vertica. The web service API can then register a tenant with one or more applications. It creates application audit event tables for the tenant and associates application tenant topics with the Kafka-Vertica Scheduler.
-
-Apache Kafka receives Audit events for an application's tenant from the client-side library and partitions them into per application, per tenant topics. The Kafka-Vertica Scheduler listens to these topics and streams the events to the tenant's application audit table in Vertica.
+Elasticsearch receives application audit events for a tenant from the client-side library and adds the application audit event to the tenant's index.
 
 ## Overview
 
-Auditing is built on Apache Kafka for the messaging of the audit events and HPE Vertica for the storage of the audit events. Both of these technologies offer high availability, throughput, scalability, and performance to the overall solution. Additionally, Vertica offers strong data analytics capabilities and comes with a pre-built Kafka integration, which can continually load data from Kafka.
-
-A Mesos/Marathon environment runs the Audit Management web service and Kafka-Vertica Scheduler components, and provides redundancy for these services.
+Auditing is built on Elasticsearch for the messaging and storage of the audit events. Elasticsearch offers high availability, throughput, scalability, and performance to the overall solution. Additionally, Elasticsearch offers strong data analytics capabilities.
 
 ### Audit Management Component Architecture
```

Comment:
This is no longer Management. Replace with Audit Service Component Architecture.